crates/sui-benchmark/src/lib.rs

```diff
@@ -519,7 +521,7 @@ impl ValidatorProxy for LocalValidatorAggregatorProxy {
             let name = *name;
             futures.push(async move {
                 client
-                    .handle_certificate_v2(certificate)
+                    .handle_certificate_v2(certificate, None)
```

Comment:
None here.

Reply:
This shouldn't be affecting the client IP of the socket connection itself as seen from the server side